cl/opinion_page/templates/recap_document.html

```diff
@@ -28,6 +28,15 @@
             src="{% static "js/save-notes.js" %}"></script>
     <script defer type="text/javascript"
             src="{% static "js/buy_pacer_modal.js" %}"></script>
+    {% if redirect_to_pacer_modal %}
+      {% include "includes/redirect_to_pacer_modal.html" %}
+      <script>
+        $(document).ready(function(){
+          $("#modal-redirect-to-pacer").modal();
+        });
```

Comment:
Seems that's not possible to activate the modal without a button action or JS, according to the [documentation](https://getbootstrap.com/docs/3.3/javascript/#modals) the modal can be activated through data attributes using an anchor tag or via JS.

The problem is that the modal not only requires to set it a CSS class to be displayed but also the `modal-open` is added to the `body` tag to display the modal background.

Reply:
Ah ha. Too bad. In that case, rather than having a little script tag here, should we just add this to our standard JS file? Maybe we can rename it as `#open-modal-on-load` and have it open a particular modal based on a data attribute that it picks up or something? That way, it could be generic and could be used on other pages eventually too?

Reply:
Great, yeah in that way we could reuse this modal that opens on load. Change is done.